tests/lints/check-external-size.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf8 -*-
+"""
+This script checks that all the external archive included in the repository are
+as small as they can be.
+"""
+from __future__ import print_function
+import os
+import sys
+import glob
+
+ROOT = os.path.join(os.path.dirname(__file__), "..", "..")
+ERRORS = 0
+
+
+# when adding new files here, make sure that they are as small as possible!
+EXPECTED_SIZES = {
+    "lzma.tar.gz": 256,
+    "mmtf-cpp.tar.gz": 439,
+    "tng.tar.gz": 317,
+    "zlib.tar.gz": 370,
+    "molfiles.tar.gz": 477,
+    "netcdf.tar.gz": 494,
+    "xdrfile.tar.gz": 41,
+    "bzip2.tar.gz": 344,
+    "pugixml.tar.gz": 549,
+    "gemmi.tar.gz": 476,
+    "fmt.tar.gz": 745,
```

Comment:
Nit: wouldn't it be nice to have things in alphabetical order?
```suggestion
    "bzip2.tar.gz": 344,
    "fmt.tar.gz": 745,
    "gemmi.tar.gz": 476,
    "lzma.tar.gz": 256,
    "mmtf-cpp.tar.gz": 439,
    "molfiles.tar.gz": 477,
    "netcdf.tar.gz": 494,
    "pugixml.tar.gz": 549,
    "tng.tar.gz": 317,
    "xdrfile.tar.gz": 41,
    "zlib.tar.gz": 370,
```